internal/common/apps.go

```diff
@@ -21,7 +21,7 @@ const (
 	DefaultDeployAppsTimeout    = 15 * time.Minute
 )
 
-func runApps() {
+func RunApps() {
```

Comment:
This change feels like a code smell. If we're now reusing this function it should be pulled out into its own package.